File: 1-week-C/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt the user for change owed, in cents
    int cents;
    do
    {
        cents = get_int("Change owed: ");
    }
    while (cents < 0);

    int coin_counter = 0;

    // Calculate how many quarters you should give customer
    coin_counter += cents / 25;
    // Subtract the value of those quarters from cents
    cents %= 25;

    // Calculate how many dimes you should give customer
    coin_counter += cents / 10;
    // Subtract the value of those dimes from remaining cents
    cents %= 10;

    // Calculate how many nickels you should give customer
    coin_counter += cents / 5;
    // Subtract the value of those nickels from remaining cents
    cents %= 5;

    // Sum the number of quarters, dimes, nickels, and pennies used
    coin_counter += cents;

    // Print total number of coins
    printf("%i\n", coin_counter);
}

File: 6-week-python/sentimental-cash/cash.py
from cs50 import get_float

def get_change():
    change = 0
    while change <= 0:
        change = int(get_float("Change: ")*100)

    count = 0
    coins = [25, 10, 5, 1]

    for coin in coins:
        count += change // coin
        change %= coin


    print(int(count))

def main():
    get_change()

main()
